rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/NSCreationServlet.java

```diff
@@ -131,17 +119,17 @@ private void createNs(HttpServletRequest request, HttpServletResponse response)
                     } else if (readingPolicyParams) {
                         addToStringParamsOrToFileParams(policyParams, policyFileParams, formFieldName, formFieldValue);
                     } else {
-                        LOGGER.warn("Unexpected param " + formFieldName);
+                        LOGGER.warn("Unexpected parameter " + formFieldName);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)